x/ibc/07-tendermint/types/header.go

```diff
@@ -37,10 +37,11 @@ func (h Header) GetHeight() exported.Height {
 	if h.Header == nil {
 		return clienttypes.ZeroHeight()
 	}
-
-	// Enforce clienttypes.Height to use 0 epoch number
-	// TODO: Retrieve epoch number from chain-id
-	return clienttypes.NewHeight(0, uint64(h.Header.Height))
+	epoch, err := clienttypes.ParseChainID(h.Header.ChainID)
+	if err != nil {
+		return clienttypes.Height{}
```

Comment:
Do we want to treat this as zero-height or just as zero-epoch (with whatever height the header itself has)?

Reply:
I think we could split this check to be more explicit.

``` go
if IsEpochFormat() {
    if epoch, err := ParseHeight(); err != nil {
        return types.ZeroHeight()
        } else {
        return epoch-<headerheight>
    }
} 

return 0-<headerheight>